src/EasyCaching.Core/IRedisCachingProvider.cs

```diff
@@ -916,14 +916,14 @@ public interface IRedisCachingProvider
         /// <param name="cacheKey"></param>
         /// <param name="members"></param>
         /// <returns></returns>
-        List<(double longitude, double latitude)?> GeoPos(string cacheKey, List<string> members);
+        List<(decimal longitude, decimal latitude)?> GeoPos(string cacheKey, List<string> members);

```

Comment:
Maybe it should be better to maintain the definition of the interface and transform it in the implementation of csredis.

If someone is using the method, it‘s not a good way to modify this one.

Reply:
There are some difference between se.redis and cs.redis. we should be united here.

Reply:
Yes, for compatibility, the interface definition should not be modified.
However, double does lose precision, refer to [#csredis/issues/226](https://github.com/2881099/csredis/issues/226), so the perfect solution should be to upgrade all double types to decimal, improve the precision, consider putting it in the next major version.

Reply:
All right, v1.x will follow it.